src/httprpc.cpp

```diff
@@ -228,6 +357,8 @@ static bool InitRPCAuthentication()
         LogPrintf("Config options rpcuser and rpcpassword will soon be deprecated. Locally-run instances may remove rpcuser to use cookie-based auth, or may be replaced with rpcauth. Please see share/rpcuser for rpcauth auth generation.\n");
         strRPCUserColonPass = GetArg("-rpcuser", "") + ":" + GetArg("-rpcpassword", "");
     }
+
+    strRPCCORSDomain = GetArg("-rpccorsdomain", "");
```

Comment:
This line should go away - it would be misleading to both end user and developer in that we don't currently support this flag.

Tagging @sidhujag for visibility.